README.md

```diff
@@ -242,3 +242,71 @@ Testing looks something like this:
 
 But in production use you'd expect it more like
    `curl <auth to your proxy> https://ssoproxy.example.com/enroll_user -d @~/.ssh/bob.pub`
+
+### Github Support
+
+Sharkey supports issuing certificates for SSH access to Github by mapping SAML identities to Github usernames through 
+the following GraphQL query:
+```
+query {
```

Comment:
I'm not sure if the README has to describe the detailed query to retrieve the mapping. The curious ones can browse the code.